Library/Homebrew/extend/os/linux/install.rb

```diff
@@ -51,5 +62,24 @@ def symlink_ld_so
       FileUtils.ln_sf ld_so, brew_ld_so
     end
     private_class_method :symlink_ld_so
+
+    def symlink_gcc_libs
+      gcc_opt_prefix = HOMEBREW_PREFIX/"opt"/OS::LINUX_PREFERRED_GCC_FORMULA
+
+      GCC_RUNTIME_LIBS.each do |library|
+        gcc_library = gcc_opt_prefix/"lib/gcc"/GCC_VERSION_SUFFIX/library
+        gcc_library_symlink = HOMEBREW_PREFIX/"lib"/library
+        # Skip if the link target doesn't exist.
+        next unless gcc_library.readable?
+
+        # Also skip if the symlink already exists.
+        next if gcc_library_symlink.readable? && (gcc_library_symlink.realpath == gcc_library)
```

Comment:
The equality comparison will always fail whenever it is made because `#realpath` will resolve into a path inside the `Cellar`.

You probably need `readlink`.

Reply:
That sounds right, I will change this.